lib/setup/setupInstall.js

```diff
@@ -1270,12 +1270,12 @@ function Install(options) {
 
     /**
      * Enumerate all meta objects of an adapter
-     * @type {(knownObjIDs: string[], adapter: string) => Promise<void>}
+     * @type {(knownObjIDs: string[], adapter: string, metaFilesToDelete: string[]) => Promise<void>}
      */
-    this.enumerateAdapterMeta = function enumerateMeta(knownObjIDs, adapter) {
+    this.enumerateAdapterMeta = function enumerateMeta(knownObjIDs, adapter, metaFilesToDelete) {
         return getObjectViewAsync('system', 'meta', {
-            startkey: adapter + '.meta',
-            endkey: adapter + '.meta\u9999'
+            startkey: adapter,
+            endkey: adapter + '\u9999'
```

Comment:
Please check if that doesn't include objects from other adapters that start with the same name, like `zwave` and `zwave2`. We had a problem a while back where deletion of one adapter actually deleted objects from two.

Reply:
Uuhhh yes ... the "." should be included ... good point!